google/auth/_cloud_sdk.py

```diff
@@ -23,20 +23,21 @@
 
 
 # The ~/.config subdirectory containing gcloud credentials.
-_CONFIG_DIRECTORY = 'gcloud'
+_CONFIG_DIRECTORY = "gcloud"
```

Comment:
It'd be nice if we did the blacken step as a separate PR.

Reply:
I'll open a separate PR for the blacken to clean up this PR

Reply:
#375 375